pkg/specgenutil/volumes.go

```diff
@@ -272,6 +272,12 @@ func parseMountOptions(mountType string, args []string) (*spec.Mount, error) {
 			if !hasValue {
 				return nil, fmt.Errorf("%v: %w", name, errOptionArg)
 			}
+			switch value {
+			case "shared", "rshared", "private", "rprivate", "slave", "rslave", "unbindable", "runbindable":
+				// Do nothing, sane value
+			default:
+				return nil, fmt.Errorf("invalid value %q for mount option", value)
```

Comment:
Note I specifically said `arg` not `value` because then it says `bind-propagation=fake` in the error and not just `fake` which may be harder to spot the error if there are a lot of mount options given

Reply:
Ah, that makes more sense